server_admin/topics/events/login.adoc

```diff
@@ -1,70 +1,111 @@
-
-In this screenshot, we're filtering only `Login` events.  Clicking the `Update` button runs the filter.
+image:{project_images}/login-events-filter.png[Login Event Filter]
 
+In this example, we filter only `Login` events. Click the *Update* button to run the filter.
 
 ==== Event Types
 
-Login events:
+*Login events:*
+
+[cols="2",options="header"]
+|===
+|Event |Description
+|Login
+|A user logs in.
+
+|Register
+|A user registers
+
+|Logout
+|A user logs out.
+
+|Code to Token
+|An application, or client, exchanges a code for a token.
+
+|Refresh Token
+|An application, or client, refreshes a token.
+
+|===
+
+*Account events:*
+
+[cols="2",options="header"]
+|===
+|Event |Description
+|Social Link
+|A user account links to a social media provider.
 
-* Login - A user has logged in.
-* Register - A user has registered.
-* Logout - A user has logged out.
-* Code to Token - An application/client has exchanged a code for a token.
-* Refresh Token - An application/client has refreshed a token.
+|Remove Social Link
+|The link from a social media account to a user account severs.
 
-Account events:
+|Update Email
+|An email address for an account changes.
 
-* Social Link - An account has been linked to a social provider.
-* Remove Social Link - A social provider has been removed from an account.
-* Update Email - The email address for an account has changed.
-* Update Profile - The profile for an account has changed.
-* Send Password Reset - A password reset email has been sent.
-* Update Password - The password for an account has changed.
-* Update TOTP - The TOTP settings for an account have changed.
-* Remove TOTP - TOTP has been removed from an account.
-* Send Verify Email - An email verification email has been sent.
-* Verify Email - The email address for an account has been verified.
+|Update Profile
+|A profile for an account changes.
 
-For all events there is a corresponding error event.
+|Send Password Reset
+|{project_name} sends a password reset email.
+
+|Update Password
+|The password for an account changes.
+
+|Update TOTP
+|The Time-based One-time Password (TOTP) settings for an account changes.
+
+|Remove TOTP
+|{project_name} removes TOTP from an account.
+
+|Send Verify Email
+|{project_name} sends an email verification email.
+
+|Verify Email
+|{project_name} verifies the email address for an account.
+
+|===
+
+Each event has a corresponding error event.
 
 ==== Event Listener
 
-Event listeners listen for events and perform an action based on that event.  There are two built-in
-listeners that come with {project_name}: Logging Event Listener and Email Event Listener.
+Event listeners listen for events and perform actions based on that event. {project_name} includes two built-in listeners, the Logging Event Listener and Email Event Listener.
+
+===== The Logging Event Listener
+The Logging Event Listener writes to a log file when an error event occurs. {project_name} enables the Logging Event Listener by default.
```

Comment:
it could have been disabled, so best to remove :{project_name} enables the Logging Event Listener by default.
How about "When the Logging Event Listener is enabled, this listener writes to a log file when an error event occurs."